### PR TITLE
[Update 1.2.1] - Updating release notes, changelog, package.json

### PR DESCRIPTION
**Pull request by bgoddar**
Updating CHANGELOG.md, package.json, and the release notes to reflect included changes to version 1.2.1

Changelog:
image

Welcome:
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 1.2.1
+* **[Feature]**: The Extension now has limited support for Visual Studion Code's Workspace Trust Feature - [PR #431](https://github.com/microsoft/vscode-edge-devtools/pull/431)
```
Comment (antross) [inline CHANGELOG.md:2]:
```suggestion
* **[Feature]**: The Extension now has limited support for Visual Studio Code's Workspace Trust Feature - [PR #431](https://github.com/microsoft/vscode-edge-devtools/pull/431)

```

**Review by antross — APPROVED** [review]
One typo we should probably fix but otherwise LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (bgoddar) [reply CHANGELOG.md]: Good catch. Fixed

**Review by mliao95 — APPROVED** [review]
Tested and LGTM

**Pull request merged**